nvtabular/workflow/workflow.py

```diff
@@ -124,7 +127,7 @@ def transform(self, dataset: Dataset) -> Dataset:
 
     def fit_schema(self, input_schema: Schema) -> "Workflow":
         schemaless_nodes = {
-            node: _get_schemaless_nodes(node.parents)
+            node: _get_schemaless_nodes(node.parents + node.dependency_nodes)
```

Comment:
This is a place an `upstream_nodes` method or something would be useful so we don't have to think about this all the time